scripts/spack/packages/mfem/package.py

```diff
@@ -655,7 +655,7 @@ def find_optional_library(name, prefix):
             all_hypre_libs = hypre.libs + hypre["lapack"].libs + hypre["blas"].libs
             hypre_gpu_libs = ""
             if "+cuda" in hypre:
-                hypre_gpu_libs = " -lcusparse -lcurand"
```

Comment:
Is this special to us or is already in mfem's spack package in the spack repo?

Reply:
It's not special to axom.  It's not yet in the spack github repo.  I'm creating a PR for spack github right now so maybe I should pull this and not jump the gun here.

Reply:
Typically if we change anything from the spack version of a package.py we mark the changes like this:

```
# AXOM EDIT START
some change here
# AXOM EDIT END
```

So when we go to update next time we don't clobber edits.

Reply:
Good idea, I will make that change.